pennylane/devices/__init__.py

```diff
@@ -52,6 +52,22 @@
     Device
     DefaultQubit
 
+Preprocessing Transforms
+------------------------
+
+The ``preprocess`` module offers several transforms that can be used in constructing the :meth:`~.devices.Device.preprocess`
+method for devices.
+
+.. currentmodule:: pennylane.devices.preprocess
+.. autosummary::
+    :toctree: api
+
+    decompose
+    validate_measurements
+    validate_device_wires
+    validate_multiprocesssing_workers
```

Comment:
Not seeing that one here:
https://xanaduai-pennylane--4659.com.readthedocs.build/en/4659/code/qml_devices.html#preprocessing-transforms

Reply:
extra s in the name. fixed locally.